.gitignore

```diff
@@ -7,5 +7,6 @@
 .DS_Store
 /build
 /captures
+/app/build
 
 app/src/main/java/org/familysearch/sampleapp/AppKeys.java
```

Comment:
After updating the .gitignore you should delete the untracked files from the project and commit the deletion

```
git rm -r --cached .
git add .
git commit -m "fixed untracked files"
```